tests/core/types/test_spend_bundle.py

```diff
@@ -0,0 +1,76 @@
+import json
+import unittest
+
+from blspy import G2Element
+
+from chia.types.spend_bundle import SpendBundle
+
+
+BLANK_SPEND_BUNDLE = SpendBundle(coin_spends=[], aggregated_signature=G2Element())
+NULL_SIGNATURE = "0xc" + "0" * 191
+
+
+class TestStructStream(unittest.TestCase):
+    def test_from_json_legacy(self):
+        JSON = (
+            """
+        {
+          "coin_solutions": [],
+          "aggregated_signature": "%s"
+        }
+        """
+            % NULL_SIGNATURE
+        )
+        spend_bundle = SpendBundle.from_json_dict(json.loads(JSON))
+        json_1 = json.loads(JSON)
+        json_2 = spend_bundle.to_json_dict(include_legacy_keys=True, exclude_modern_keys=True)
+        assert json.loads(JSON) == spend_bundle.to_json_dict()
```

Comment:
This is failing flake8 because of unused variables.  Should this be `assert json_1 == json_2`?